Python 3.7.3 (default, Apr  3 2019, 05:39:12) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license()" for more information.
>>> 
KeyboardInterrupt
>>> Dict=['Name'='lilly']
SyntaxError: invalid syntax
>>> Dict=['Name','lilly']
>>> Dict
['Name', 'lilly']
>>> Dict=['Name:''lilly']
>>> Dict
['Name:lilly']
>>> Dict=['Name:''lilly','Age:',21]
>>> Dict
['Name:lilly', 'Age:', 21]
>>> Dict=['Name:''lilly','Age:',21,'Place:''Bangalore']
>>> Dict
['Name:lilly', 'Age:', 21, 'Place:Bangalore']
>>> Dict.copy
<built-in method copy of list object at 0x7ff7b52bf808>
>>> Dict.values
Traceback (most recent call last):
  File "<pyshell#10>", line 1, in <module>
    Dict.values
AttributeError: 'list' object has no attribute 'values'
>>> Dict.values()
Traceback (most recent call last):
  File "<pyshell#11>", line 1, in <module>
    Dict.values()
AttributeError: 'list' object has no attribute 'values'
>>> Dict.append()
Traceback (most recent call last):
  File "<pyshell#12>", line 1, in <module>
    Dict.append()
TypeError: append() takes exactly one argument (0 given)
>>> Dict.values ()
Traceback (most recent call last):
  File "<pyshell#13>", line 1, in <module>
    Dict.values ()
AttributeError: 'list' object has no attribute 'values'
>>> Dict=['Name:''lilly','Age:',21,'Place:''Bangalore']
>>> Dict
['Name:lilly', 'Age:', 21, 'Place:Bangalore']
>>> print (Dict['lilly'])
Traceback (most recent call last):
  File "<pyshell#16>", line 1, in <module>
    print (Dict['lilly'])
TypeError: list indices must be integers or slices, not str
>>> print (Dict['Name'])
Traceback (most recent call last):
  File "<pyshell#17>", line 1, in <module>
    print (Dict['Name'])
TypeError: list indices must be integers or slices, not str
>>> print(Dict[21])
Traceback (most recent call last):
  File "<pyshell#18>", line 1, in <module>
    print(Dict[21])
IndexError: list index out of range
>>> Dict = {'Tim': 18,'Charlie':12,'Tiffany':22,'Robert':25}
>>> Dict
{'Tim': 18, 'Charlie': 12, 'Tiffany': 22, 'Robert': 25}
>>> print (Dict['Tiffany'])
22
>>> print (Dict['Tim'])
18
>>> Dict = ['Tim': 18,'Charlie':12,'Tiffany':22,'Robert':25]
SyntaxError: invalid syntax
>>> 
KeyboardInterrupt
>>> Dict = {'Tim': 18,'Charlie':12,'Tiffany':22,'Robert':25}
>>> Boys = {'Tim': 18,'Charlie':12,'Robert':25}
>>> Boys
{'Tim': 18, 'Charlie': 12, 'Robert': 25}
>>> Dict
{'Tim': 18, 'Charlie': 12, 'Tiffany': 22, 'Robert': 25}
>>> print (Dict[12])
Traceback (most recent call last):
  File "<pyshell#28>", line 1, in <module>
    print (Dict[12])
KeyError: 12
>>> print (Boys[12])
Traceback (most recent call last):
  File "<pyshell#29>", line 1, in <module>
    print (Boys[12])
KeyError: 12
>>> print (Boys{12})
SyntaxError: invalid syntax
>>> Dict.update({"Sarah":9})
>>> Dict
{'Tim': 18, 'Charlie': 12, 'Tiffany': 22, 'Robert': 25, 'Sarah': 9}
>>> Dict.update({"Sarah":9})
>>> Dict
{'Tim': 18, 'Charlie': 12, 'Tiffany': 22, 'Robert': 25, 'Sarah': 9}
>>> print (Dict['Robert'])
25
>>> Dict.copy
<built-in method copy of dict object at 0x7ff7b4d8bcf0>
>>> Dict.values()
dict_values([18, 12, 22, 25, 9])
>>> Dict.pop()
Traceback (most recent call last):
  File "<pyshell#38>", line 1, in <module>
    Dict.pop()
TypeError: pop expected at least 1 arguments, got 0
>>> Dict.pop('Tim')
18
>>> Dict.keys()
dict_keys(['Charlie', 'Tiffany', 'Robert', 'Sarah'])
>>> Dict.items()
dict_items([('Charlie', 12), ('Tiffany', 22), ('Robert', 25), ('Sarah', 9)])
>>> Dict.get('Tim')
>>> Dict
{'Charlie': 12, 'Tiffany': 22, 'Robert': 25, 'Sarah': 9}
>>> Dict.restore('Tim')
Traceback (most recent call last):
  File "<pyshell#44>", line 1, in <module>
    Dict.restore('Tim')
AttributeError: 'dict' object has no attribute 'restore'
>>> Dict.restore()
Traceback (most recent call last):
  File "<pyshell#45>", line 1, in <module>
    Dict.restore()
AttributeError: 'dict' object has no attribute 'restore'
>>> len(Dict)
4
>>> Dict.copy()
{'Charlie': 12, 'Tiffany': 22, 'Robert': 25, 'Sarah': 9}
>>> Dict.fromkeys()
Traceback (most recent call last):
  File "<pyshell#48>", line 1, in <module>
    Dict.fromkeys()
TypeError: fromkeys expected at least 1 arguments, got 0
>>> Dict.fromkeys(Dict,Boys)
{'Charlie': {'Tim': 18, 'Charlie': 12, 'Robert': 25}, 'Tiffany': {'Tim': 18, 'Charlie': 12, 'Robert': 25}, 'Robert': {'Tim': 18, 'Charlie': 12, 'Robert': 25}, 'Sarah': {'Tim': 18, 'Charlie': 12, 'Robert': 25}}
>>> Dict.setdefault(NAMES,VALUES)
Traceback (most recent call last):
  File "<pyshell#50>", line 1, in <module>
    Dict.setdefault(NAMES,VALUES)
NameError: name 'NAMES' is not defined
>>> Dict.setdefault(keyname, value)
Traceback (most recent call last):
  File "<pyshell#51>", line 1, in <module>
    Dict.setdefault(keyname, value)
NameError: name 'keyname' is not defined
>>> Dict.setdefault(Name,Age)
Traceback (most recent call last):
  File "<pyshell#52>", line 1, in <module>
    Dict.setdefault(Name,Age)
NameError: name 'Name' is not defined
>>> Dict.setdefault(Name,Tim)
Traceback (most recent call last):
  File "<pyshell#53>", line 1, in <module>
    Dict.setdefault(Name,Tim)
NameError: name 'Name' is not defined
>>> Dict.setdefault(Name,Sarah)
Traceback (most recent call last):
  File "<pyshell#54>", line 1, in <module>
    Dict.setdefault(Name,Sarah)
NameError: name 'Name' is not defined
>>> Dict.setdefault('Name','Sarah')
'Sarah'
>>> Dict.popitem('Name','Robert')
Traceback (most recent call last):
  File "<pyshell#56>", line 1, in <module>
    Dict.popitem('Name','Robert')
TypeError: popitem() takes no arguments (2 given)
>>> 
KeyboardInterrupt
>>> Dict.popitem('Name','Sarah')
Traceback (most recent call last):
  File "<pyshell#57>", line 1, in <module>
    Dict.popitem('Name','Sarah')
TypeError: popitem() takes no arguments (2 given)
>>> Dict.clear()
>>> Dict
{}
>>> Dict = {'Tim': 18,'Charlie':12,'Tiffany':22,'Robert':25}
>>> Dict
{'Tim': 18, 'Charlie': 12, 'Tiffany': 22, 'Robert': 25}
>>> Dict.popitem()
('Robert', 25)
>>> 
